### DISCUSSION TITLE
go2 training help

### DISCUSSION THREAD

**Discussion by CallumGracie** [abizovnuralem/go2_ros2_sdk#69] [Q&A]
I'm trying to wrap my head around how to train the go2 using this repo. Is it a matter of amending the run sim method to call learn methods and get the a lot robots in the environment that don't have key based events? Or have I overlooked a easier way to achieve this in the current repo ? 

If you could point me in the right direction or a similar example that would be great. For context our real go2 uses the go2 ros2 sdk and it's our intention to use the RL env in isaac to improve the movement across undulating terrains, different surfaces and then look at more complex tasks like detection of stairs to adjust the movements without switching modes.

**Comment by abizovnuralem** [comment]
Great plans! For the RL I suggest you start with official docs and scripts that have pre-defined envs to train GO2 and other robots. The current repo and go2_omniverse are the examples of simulation + ros2 bridge
